packages/acceptance-tests/pkg-tests-fixtures/packages/node-gyp-1.0.0/node-gyp.js

```diff
@@ -0,0 +1,4 @@
+const fs = require('fs');
+
+
+fs.writeFileSync('build.node', `${process.cwd()}/build.node`);
```

Comment:
```suggestion
fs.writeFileSync('build.node', `${process.cwd()}${path.sep}build.node`);
```

(you'll also need to require `path`)

Reply:
This is the cause of the Windows failure btw

Reply:
Thanks, didn't see that

Reply:
It should be good now, waiting for CI